### ISSUE TITLE
Subscript out of bounds error in qqcomp

### ISSUE THREAD

**Issue by frederikziebell** [aursiber/fitdistrplus#4]
Running 
```
set.seed(1)
x <- rnorm(10^5)
fit <- fitdist(x,"norm")
qqcomp(fit)
```
gives `Error in fittedquant[, i] : subscript out of bounds`. The error does not appear if only 10^4 numbers are simulated, so I guess it's some sort of edge-case.

**Comment by dutangc** [comment]
Indeed an error is raised yet the graphic is plotted. We will investigate asap.

**Comment by frederikziebell** [comment]
Thanks for going after this. Yes, a graphic is plotted but the identity line is missing.

**Comment by aursiber** [comment]
Bug fixed in 8056cdc2ada2563253c9b54fb7749fd607356d84
